server/database/schema.sql

```diff
@@ -796,6 +796,21 @@ CREATE TABLE IF NOT EXISTS toolset_prompts (
 CREATE UNIQUE INDEX IF NOT EXISTS toolset_prompts_toolset_id_prompt_name_key
 ON toolset_prompts (toolset_id, prompt_name);
 
+CREATE TABLE IF NOT EXISTS mcp_install_page_metadata (
```

Comment:
Thoughts on generalizing this more into mcp_metadata. I think we will have more reasons to attach metadata to a specific mcp going forward. 

Not reason it needs to be isolated to install

Reply:
I think I like that. Was a little hesitant about introducing too broad of an abstraction but if you're game I certainly am.